### DISCUSSION TITLE
name.com

### DISCUSSION THREAD

**Discussion by yusufyr7** [community/community#116429] [GitHub Education]
### Hi everyone,

öğrenciliğimi doğruladım fakat name.com Bu GitHub öğrenci kimlik bilgileri geçerli değil. Daha fazla bilgi için lütfen [Github Öğrenci Geliştirici Paketi makalesine](https://help.github.com/en/articles/applying-for-a-student-developer-pack) bakın. bunu demekte

**Comment by queenofcorgis** [comment]
Hi @yusufyr7 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
